runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordPlugin.java

```diff
@@ -149,7 +156,6 @@ protected void shutDown() throws Exception
 	{
 		clientToolbar.removeNavigation(discordButton);
 		resetState();
-		partyService.changeParty(null);
```

Comment:
The party stuff largely doesn't function without the discord plugin (such as join requests don't work, avatars don't work) and so this was here to remove the player from a party if it was shut off. Why is it removed?

Reply:
Because there are other plugins that can be used to join a party, like easy party does it with a partyId directly. Someone can be in a party regardless of discord.
It was necessary before because there was no "leave party" button. I added one.

Reply:
But there was leave party button, right click on the in game interface

Reply:
Yes that's right. But the interface was also toggleable and did not appear when you were the only party member. The Leave party button is now always there when you are in a party.

Reply:
I really dont see how is that related. You are breaking the party by disabling Discord plugin, so why are you not leaving the party as well. If you dont want to interfere with other party plugins that exist I guess? then handle it like that and not completely ignore the issue.

Reply:
I don't understand the issue. What do you mean by "handle it like that"?
As you said, I don't want to interfere with other party plugins.
Also, there might be people that enable Discord, join a party and shut discord down. I don't see why Discord has to force someone out of a party when you can decide when to leave by clicking a "Leave party" button. What am I missing here?